docs/DeveloperGuide.md

```diff
@@ -314,9 +314,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `DonnaFin` application and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**UC01: Adding a contact to DonnaFin**
+
+**MSS**
+
+1.  User types command to add contact along with the relevant details
```

Comment:
User chooses to add contact with relevant details. 

Rmb: We wanna leave out how its done and only mention what is done

Reply:
Please change this everywhere

Reply:
Okay